test/data/stages/pki.update-ca-trust/diff.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Maybe I'm missing something, but wouldn't we expect some sort of tree diff here and if we won't, what is it we are exactly testing?

Reply:
Ah messed up git commit, indeed there should be a diff, let me generate it again.